content/articles/data-viz-chartify/index.md

```diff
@@ -0,0 +1,366 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /data-viz-chartify/
+title: Data Visualization with Chartify
+description: Chartify is an open-source data visualization library from Spotify that makes it easy for data analysts to create charts and graphs. This article gives a brief introduction to this technology. 
+author: benedict-ifeanyi-iheagwara
+date: 2022-01-18T00:00:00-00:20
+topics: [API]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/data-viz-chartify/hero.png
+    alt: Data visualisation with Chartify
+---
+Although Chartify isn�t the new kid on the block, many data scientists are still unfamiliar with it. That�s a bummer, considering that it was developed to make data visualization easier for data scientists and is more 'intuitive' than other Python charting libraries. Data visualization is an important aspect of data analytics since it allows you to communicate information to shareholders in a visually appealing and understandable way.
```

Comment:
```suggestion
Although Chartify isn't the new kid on the block, many data scientists are still unfamiliar with it. That's a bummer, considering that it was developed to make data visualization easier for data scientists and is more 'intuitive' than other Python charting libraries. Data visualization is an important aspect of data analytics since it allows you to communicate information to shareholders in a visually appealing and understandable way.
```

Reply:
Ops I forgot  to highlight this minor change @Bennykillua